src/ducks/SANCharts/ChartActiveMetrics.js

```diff
@@ -12,6 +12,7 @@ import styles from './ChartActiveMetrics.module.scss'
 const ChartActiveMetrics = ({
   activeEvents,
   activeMetrics,
+  alwaysShowingMetrics = [],
```

Comment:
and just for clarity - could you explain when we need this prop - `alwaysShowingMetrics`? In what cases

Reply:
ChartWidget can't show triggered signals. They passed outside of widget as ReferenceDots with Y-position related to Y-axis of triggered metrics. And user can open this chart and unselect metric Then chart widget will remove Y-axis for this metrics. And chart will crash with error, because Reference dots has reference to removed Y-axis.
For this case I mark metric of triggered signals as 'alwaysShowing'. It mean, that only on Smart Chart of TriggerForm this metric can't me removed from chart.

Reply:
Also we can say inside of widget that metric was removed...But it is't working, because rechart can't change reference to removed Y-axis and chart will be crashed with error to undefined Y-axis.